.github/workflows/lint_and_suggest_pr.yml

```diff
@@ -13,3 +13,13 @@ jobs:
         with:
           mode: suggest
           lint-bib-file: true
+      - name: All Mathlib files have lean or md extension
+        run: |
+          exc="$(git ls-files 'Mathlib/*' | grep -v "\.\(lean\|md\)$" || true)"
```

Comment:
```suggestion
          exc="$(git ls-files 'Mathlib/*' | grep --invert-match "\.\(lean\|md\)$" || true)"

Reply:
I normally like extended names, but in the case of `-v` I actually had no idea what the extended name was, so I personally would find this less clear and would wonder "This is surely not the same as `-v`!"...  :upside_down_face: